### ISSUE TITLE
Lighting is different then the original

### ISSUE THREAD

**Issue by Asphaltian** [MaximumADHD/Source2Roblox#32]
This issue is consistent in every source map I import to Roblox Studio. I'm trying to import bm_c1a0a (a Black Mesa map), but when I try to do so, the map is extremely dark.  I saw your d1_trainstation_01 video, and the lighting is completely different in your video. Am I missing something? Here are some screenshots:

Original:
![download](https://user-images.githubusercontent.com/86578963/140603974-da62ebb4-e548-476a-aa88-23dc47b85933.jpg)
ROBLOX version:
![Screenshot 2021-11-06 150754](https://user-images.githubusercontent.com/86578963/140603987-3d8f7d27-ac68-45a2-960a-7cd8463eb1fb.png)

**Comment by MaximumADHD** [comment]
I approximate the lighting as best as I can but the sheer variety of parameters and shaders that Source’s lighting provides is difficult to match 1:1 in Roblox.

Black Mesa has its own additional lighting engine with completely separate entities which I’m not accounting for at this time. I will look into whether I can support them soon.

**Comment by Gamer3789** [comment]
It seems like you've never heard the phrase "Nothing is perfect", Asphaltian.

**Comment by Asphaltian** [comment]
> It seems like you've never heard the phrase "Nothing is perfect", Asphaltian.

Obviously, I lowered my expectations as I don't expect a 1:1 replica of the original. I'm just asking if the lighting could be improved
